src/PKSim.Infrastructure/ORM/Repositories/OntogenyRepository.cs

```diff
@@ -136,8 +136,23 @@ public DistributedTableFormula DataRepositoryToDistributedTableFormula(DataRepos
          }
          else
          {
-            meanColumn = valueColumns.Single(x => x.RelatedColumns.Any());
-            deviationColumn = valueColumns.Single(x => !x.RelatedColumns.Any());
+            //the main column is the one that has related column
+            var mainColumn = valueColumns.Single(x => x.RelatedColumns.Any());
+            var relatedColumn = mainColumn.RelatedColumns.Single();
+            var auxiliaryType = relatedColumn.DataInfo.AuxiliaryType;
+
+            //this is the way data repository are created when coming from the PKSim DB
+            if (auxiliaryType == AuxiliaryType.GeometricMeanPop)
```

Comment:
took me a while to figure this out but in short. We have TWO TYPE of tables. The one imported from importer and the one defined for by PKSim. They used different conventions. (As discussed somewhere else by design)